vmdb/spec/requests/api/providers_spec.rb

```diff
@@ -33,7 +33,7 @@
     {
       "type"      => "EmsRedhat",
       "name"      => "sample rhevm",
-      "port"      => 5000,
```

Comment:
@abellotti how do you want to do this?  I'd like to re-enable as many of these tests as possible but it looks like we really need to understand why this test needs the port as a string for rails 4.2, while either a String or Fixnum was fine for rails 3.2.  Should I keep this change for now and open an issue to research this later?